Classes/System/Dispatcher.php

```diff
@@ -26,31 +26,59 @@
  ***************************************************************/
 
 use Aoe\Restler\System\Restler\Builder as RestlerBuilder;
+use Aoe\Restler\System\TYPO3\Loader;
+use Luracast\Restler\Routes;
+use Psr\Http\Message\ResponseInterface;
+use Psr\Http\Message\ServerRequestInterface;
+use Psr\Http\Server\MiddlewareInterface;
+use Psr\Http\Server\RequestHandlerInterface;
+use TYPO3\CMS\Core\Http\Response;
+use TYPO3\CMS\Extbase\Object\ObjectManager;
 
 /**
  * @package Restler
  */
-class Dispatcher
+class Dispatcher implements MiddlewareInterface
 {
     /**
      * @var RestlerBuilder
      */
     private $restlerBuilder;
 
-    /**
-     * @param RestlerBuilder $restlerBuilder
-     */
-    public function __construct(RestlerBuilder $restlerBuilder)
+    public function __construct(ObjectManager $objectManager=null)
     {
-        $this->restlerBuilder = $restlerBuilder;
+        if (!$objectManager){
+            $objectManager = new ObjectManager();
```

Comment:
Why not GeneralUtility::makeInstance(ObjectManager::class) ?
Think that will give us an auto-load advantaged